roles/nvidia-dgx/tasks/ubuntu.yml

```diff
@@ -1,4 +1,9 @@
 ---
+- name: Check for Ubuntu version
+  fail:
+    msg: "Role supports Ubuntu 18.04 only"
+  when: ansible_distribution_version != "18.04"
+
```

Comment:
Just a suggestion for future-proofing: we may want to re-name the `ubuntu.yml` file to something like `ubuntu-18.04.yml` and put this check in `main.yml`, i.e.

```
- name: ubuntu tasks
  include_tasks: ubuntu-18.04.yml
  when:
    - ansible_distribution == 'Ubuntu'
    - ansible_distribution_version == "18.04"
```

Reply:
Done!